cassandra-four-zero-bridge/src/main/java/org/apache/cassandra/spark/reader/MapBuffer.java

```diff
@@ -17,28 +17,21 @@
  * under the License.
  */
 
-package org.apache.cassandra.bridge;
+package org.apache.cassandra.spark.reader;
 
-public class RangeTombstone
-{
-    public final Bound open;
-    public final Bound close;
+import org.apache.cassandra.db.rows.Cell;
 
-    public RangeTombstone(Bound open, Bound close)
+public class MapBuffer extends ComplexTypeBuffer
+{
+    MapBuffer(int cellCount)
     {
-        this.open = open;
-        this.close = close;
+        super(cellCount, cellCount * 2);
```

Comment:
Can you please add a comment about why double cell count is used here?

Reply:
Yes, it's because 1 for the key & value. I'll add a comment.